pilot/pkg/bootstrap/certcontroller.go

```diff
@@ -133,6 +135,17 @@ func (s *Server) initDNSCerts() error {
 				}()
 				return nil
 			})
+		} else if _, err := os.Stat(path.Join(LocalCertDir.Get(), ca.IstioGenerated)); err == nil {
+			// TODO(jaellio): Modify to read secret data from file.
+			log.Infof("Found cert at %v, but is istio-generated; self-signed cert is used", fileBundle.SigningKeyFile)
```

Comment:
not fileBundle.SigningKeyFile.. but istioGenerated

Reply:
Are you suggesting that I pass the istio generated file path as the argument instead of the signing key file path?